package.json

```diff
@@ -36,6 +36,9 @@
     "build:bundle": "cross-env NODE_ENV=production webpack -p --progress --bail --mode production",
     "build:bundle:dev": "webpack --progress --bail --mode development",
     "build:electron": "yarn clean && yarn build && yarn install:electron && build -wml --ia32 --x64",
+    "build:electron:linux": "yarn build && build -l --ia32 --x64",
```

Comment:
On macOS and Linux, we only support and distribute 64-bit, so `ia32` can be removed for those lines.